intro_to_git/Git_Workshop_v4.ipynb

```diff
@@ -3,9 +3,7 @@
   {
```

Comment:
<p>This looks good, hopefully it's a helpful note. </p>

 _Reply via <a href='https://app.reviewnb.com/georgetown-analytics/XBUS-501-01.Software-Engineering-for-Data/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>